### PR TITLE
Coverage test and refactor

### PR DESCRIPTION
**Pull request by pospieszynski**
@gotar

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.idea/
```
Comment (wafcio) [inline .gitignore]: move files genetrated by editors to global .gitignore: https://help.github.com/articles/ignoring-files/

File: lib/product.rb [hunk 2]
```diff
@@ -20,4 +20,6 @@ def arguments_validation name, price
     raise ArgumentError if name == nil || price == nil
     raise TypeError unless name.is_a?(String) && price.is_a?(Numeric)
   end
-end
\ No newline at end of file
+end
+
+
```
Comment (wafcio) [inline lib/product.rb]: Remove this empty lines, one empty line at the end of file is enough.

File: spec/spec_helper.rb [hunk 3]
```diff
@@ -1,3 +1,7 @@
+require 'simplecov'
```
Comment (wafcio) [inline spec/spec_helper.rb:1]: Everywhere you have double quotes, but here you have single quotes.

File: Gemfile [hunk 4]
```diff
@@ -1,5 +1,6 @@
 source "https://rubygems.org"
 
 group :test do
+  gem "simplecov"
   gem "rspec"
 end
```
Comment (wafcio) [inline Gemfile:6]: Set up your editor to add empty line at the end of file

File: lib/shop_basket.rb [hunk 5]
```diff
@@ -23,11 +23,11 @@ def remove name
...
 
-  def price_brutto
-    price_netto * 1.23
+  def price_gross
+    price_net * 1.23
   end
 
-  def price_netto
+  def price_net
     basket.reduce(0) { |sum, n| sum + n.price }
```
Comment (gotar) [inline lib/shop_basket.rb:31]: `basket.reduce(0, &:price)`

**Pull request closed without merging**